components/Layout.js

```diff
@@ -16,12 +19,49 @@ const Layout = props => (
     <center>
       <NavigationBar />
     </center>
+    <div style=
```

Comment:
Move this part to index.js as this layout is used everywhere. Layout.js is used to create new pages. If you see the about page then you will see. right now it has broken down.

This section is specific to the page so it should be used only in index.js.
Moreover, if you create a new component called HomeMessage.js for this part and add it to the index.js then it will be amazing. then we can use it on other pages also if we need.

Reply:
Ok, I'll work on it.